src/drivers/distance_sensor/tfmini/tfmini.cpp

```diff
@@ -80,51 +81,51 @@
 # error This requires CONFIG_SCHED_WORKQUEUE.
 #endif
 
-// designated SERIAL4/5 on Pixhawk
-#define TFMINI_DEFAULT_PORT		"/dev/ttyS6"
+#define NAME "tfmini"
+#define DEVICE_PATH "/dev/" NAME
```

Comment:
These are different things.
`/dev/tfmini` is the px4 device driver (created by this module).
`/dev/ttyS6` or whatever needs to be the physical serial port.

Reply:
I think this is the case, isn't it? It is not directly replacing it

Reply:
Sorry you're right, I should really stop trying to view diffs on my phone. There's no default port then?

Reply:
hehe, no worries :) Yes, there's no default port anymore. I think it makes no sense to have one if it's different on every board

Reply:
What's wrong with defaulting to RANGE_FINDER0_DEVICE_PATH and letting the command line override?